pkg/state/script_caller.go

```diff
@@ -288,22 +288,22 @@ func (a *scriptCaller) invokeFunction(tree *ride.Tree, tx proto.Transaction, inf
 
 	r, err := ride.CallFunction(env, tree, functionName, functionArguments)
 	if err != nil {
-		return nil, errors.Wrapf(err, "invocation of transaction '%s' failed", txID.String())
+		if err2 := a.appendFunctionComplexity(ride.EvaluationErrorSpentComplexity(err), scriptAddress, functionName, defaultFunction, info); err2 != nil {
+			return nil, err
+		}
```

Comment:
I think `return nil, err` is a mistake. `return nil, err2` is correct (btw, `err2` doesn't looks like good naming)